lib/src/components/NxBinaryDonutChart/__tests__/NxDonutChart.test.tsx

```diff
@@ -0,0 +1,70 @@
+  it('renders an svg with the expected properties', function() {
+    const getShallowComponent = enzymeUtils.getShallowComponent(NxBinaryDonutChart, { percent: 90 });
+
+    expect(getShallowComponent()).toMatchSelector('svg');
+    expect(getShallowComponent()).toHaveProp('viewBox', '-100 -100 200 200');
+    expect(getShallowComponent()).toHaveClassName('.nx-binary-donut-chart');
+  });
+
+  it('renders only the background circle with the expected properties at zero percent', function() {
+    const getShallowComponent = enzymeUtils.getShallowComponent(NxBinaryDonutChart, { percent: 0 });
+    const circles = getShallowComponent().find('circle');
+    const paths = getShallowComponent().find('path');
+
+    expect(circles.length).toBe(1);
+    expect(circles.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
+    expect(circles.at(0)).toHaveProp('strokeWidth', 50);
+    expect(circles.at(0)).toHaveProp('r', 75);
+    expect(paths.length).toBe(0);
+  });
+
+  it('renders only a background circle with the expected properties when percent is 0', function() {
+    const getShallowComponent = enzymeUtils.getShallowComponent(NxBinaryDonutChart, { percent: 0 });
+    const circles = getShallowComponent().find('circle');
+    const paths = getShallowComponent().find('path');
+
+    expect(circles.length).toBe(1);
+    expect(circles.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
+    expect(circles.at(0)).toHaveProp('strokeWidth', 50);
+    expect(circles.at(0)).toHaveProp('r', 75);
+    expect(paths.length).toBe(0);
+  });
+
+  it('renders only an arc circle with the expected properties when percent is 100', function() {
+    const getShallowComponent = enzymeUtils.getShallowComponent(NxBinaryDonutChart, { percent: 100 });
+    const circles = getShallowComponent().find('circle');
+    const paths = getShallowComponent().find('path');
+
+    expect(circles.length).toBe(1);
+    expect(circles.at(0)).toHaveClassName('.nx-binary-donut-chart__arc');
+    expect(circles.at(0)).toHaveProp('strokeWidth', 50);
+    expect(circles.at(0)).toHaveProp('r', 75);
+    expect(paths.length).toBe(0);
+  });
+
+  it('renders a background circle and arc path with expected properties when 0 < percent < 100', function() {
+    const getShallowComponent = enzymeUtils.getShallowComponent(NxBinaryDonutChart, { percent: 50 });
+    const circles = getShallowComponent().find('circle');
+    const paths = getShallowComponent().find('path');
+
+    expect(circles.length).toBe(1);
+    expect(circles.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
+    expect(circles.at(0)).toHaveProp('strokeWidth', 50);
+    expect(circles.at(0)).toHaveProp('r', 75);
+    expect(paths.length).toBe(1);
+    expect(paths.at(0)).toHaveClassName('.nx-binary-donut-chart__arc');
+    expect(paths.at(0)).toHaveProp('d', 'M 0 -75 A 75 75 0 0 0 ' + (75 * Math.cos(1.5 * Math.PI))
+        + ' ' + (-75 * Math.sin(1.5 * Math.PI)));
```

Comment:
These trig calcs should come out to 0 and 1 respectively

Reply:
> These trig calcs should come out to 0 and 1 respectively

I am checking this and they are not, also, AFAIK the values should be the same for any percentage >0 and <100 right? I was testing with 72 and it's different, so maybe there's something going on

Reply:
> I am checking this and they are not (-1.3777276490407723e-14 and 75)

Sorry, I meant they should come out to 0 and `r`, which they basically do.  Unfortunately, what with π being an irrational number, there are going to be rounding errors which is where that decimal is coming from.

> also, AFAIK the values should be the same for any percentage >0 and <100 right?

No.  These values define where the arc stops.  That they are different for different percentage values is the core functionality of the component.